flask_monitoringdashboard/database/median_group.py

```diff
@@ -0,0 +1,23 @@
+from flask_monitoringdashboard.database import FunctionCall
+from numpy import median
+
+
+def get_median_group(db_session, *where):
+    """ Return the median for a specific endpoint within a certain time interval.
+    If date_from is not specified, all results are counted
+    :param db_session: session for the database
+    :param where: additional where clause
+    """
+    result = db_session.query(FunctionCall.endpoint, FunctionCall.execution_time). \
+        filter(*where).order_by(FunctionCall.endpoint).all()
+    # result is now a list of tuples per request.
+    combine = {}
```

Comment:
Sorry, I just want to notice that you can use defaultdict pattern here. It is more efficient (without multiple lookup via *in*) and readable
```
from collections import defaultdict
combine = defaultdict(list)
for key, value in result:
    combine[key].append(value)
```